doc/examples/142-https-pem-cert/config.json

```diff
@@ -0,0 +1,17 @@
T\nRGVmYXVsdCBDb21wYW55IEx0ZDEsMCoGA1UEAwwjbW9jay1zZXJ2ZXItaHR0cHMt\nc2VsZi1zaWduZWQtcHJveHkwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIB\nAQDDf+vS3gnOU8dibGh1VTrEPry172oWPEp4Ybn31h5+NWpnmKWufGRc3HeZ0\/cJ\nJ4V4HQmiCXc6yGxStx3EMaGT11DzSvbguZChSj+DxckFgxuox4wL7bOzScM\/yWkL\nBK21wxLl92RiKlgBvdt3bDyJbOraxGSKBNRBBLM6Wh1JWc06YxF44e0T3B7H\/Qtn\nWPXmsi1YPbYAxlkeoeJ6WNwfzEuLMnFf\/fWgL2TgFwLIfhqOzlyU2CyHSAhOX3Ny\n5x3wIMxdT51MW601iCKhdluN6aNM0lTgHNAiz7RD0ZFtqWDN0XvOzTU+7jxA0926\nqtk18g9XU9R3WDNw1p6FNbdrAgMBAAEwDQYJKoZIhvcNAQELBQADggIBAFWQJM3l\nWimy7DYKn3Dy1RH9Pd4mIT0TJ4V1T\/eszbWB7YRDdHuYyf7Sk0QDZPLMqR2sYxMc\nShdrQpOgU\/o\/iIkhwAjW6LzbuiQJULd74u3J08RPAEpSSJFJwQD4ZU2B7nhoqxft\noxUzoytfUSnsMA2EUJFYsPvkygyCihudNR+44ONq771W9HfvFNH3YdQyxjI\/RtwT\nCiYdxQFiS5UvWCBbJAVmRWLm1gnJbG+MYBye4mGfyiZ3iMq\/Wnb++favC8jNDYSO\nokha+LQBpscQZ3fbn5srh7uQGXOkQjT8gsd31neEFfXbVlrnUsHAXMHHgUgNrF+k\nPVp0321fzmvwxGwjB4\/jHd+1ImDwcDJ6LseKwtbriN5EDbwEV\/mIME2\/MjFfqFhV\nM3AkhgW\/sY4KvAD+FZ77U3xR4iIQgknjAPEYgqKDHF0h0295fwhRl7wmw\/nO6Z8e\nIHP+YOH7gcdYskr2cnXXD9J2z0T3\/VZUyZjnrmVqCdWq4hggea1qKYw6qHbtuETV\n9CCxLE+LTrFfYHHkQI2Jg8E+NDApjOodFoSaLphTzK8FZoyn9ttrR8UcFFMW95nt\n9POau3WtvfYvAve0h4UYj9ZMotNnTFWyhmgn9gu6gvuyVfvUKJlsW+KQbJ8bJraN\n0SLVT2nUF\/WUpJde3jvWs+OtQCQEUZds5qfy\n-----END CERTIFICATE-----\n-----BEGIN RSA PRIVATE KEY-----\nMIIEpgIBAAKCAQEAw3\/r0t4JzlPHYmxodVU6xD68te9qFjxKeGG599YefjVqZ5il\nrnxkXNx3mdP3CSeFeB0Jogl3OshsUrcdxDGhk9dQ80r24LmQoUo\/g8XJBYMbqMeM\nC+2zs0nDP8lpCwSttcMS5fdkYipYAb3bd2w8iWzq2sRkigTUQQSzOlodSVnNOmMR\neOHtE9wex\/0LZ1j15rItWD22AMZZHqHieljcH8xLizJxX\/31oC9k4BcCyH4ajs5c\nlNgsh0gITl9zcucd8CDMXU+dTFutNYgioXZbjemjTNJU4BzQIs+0Q9GRbalgzdF7\nzs01Pu48QNPduqrZNfIPV1PUd1gzcNaehTW3awIDAQABAoIBAQC0TdeWbFgI5fW+\nzZLtnASOF\/p9pWhnR4W7Tw5wCBepeORA1WUA7xK1HlRkGDz\/QWb0Z3uJ6UcAfPaC\ntwYJ+2g07OFo8VuN2TcNumenxZNntQqeeCOuoLUahs4\/lyd9FPyM5XQZijgX3Uu7\nl3nAw591c3GLAJRlQpzrtkUCbTgTe9TlIzE6YrNlMzQyKNCRFOSXgOW1m0hq6u6k\nC\/ebqyeCUemgVviCkfQFuPbY\/YSa9OTrqaeNEdgIug4aRJ4x8JCuVR\/zeKSyv2oX\nk\/Y+7FJA5hv5vm7LjsT14xuGt41oiHfRnF1jso8nZnd\/vIMII4btF+FfMZhdphle\noL\/71rgxAoGBAOxspcqkxjuLSMJ1hkZKdi8akwtqDKqyBwIBKPhC4vO09pWer8sR\n2iQuywQZu+oi\/aVcFENAJuTldoNQW4wgjvUEa34HsBC9KHBCPufY\/iB+Z2ZieOX4\nL+pn9QNZfc1DIk6aoGbSCmF+HSD\/DMnt4fTRqvZp1uLwWgWS7vEDTVSJAoGBANOv\n0OBkn9jjo2k+JqRFowoaxVUxivlmRggvNA6sF3deTgJAyszblU2UWvgDNEfUvcQD\nI80BhTbdsZ+bKYzRPw3U4m7Ug17JKlCV7eK2d4F13BnSUfoFlEhBT5LS13PhLfg3\nKeuAWuDkPTG2T22sGiBzg2UM1ZitGMXsQzGPcxdTAoGBAOBUFtu4jP9ioZ8htE5t\nyMXZCT0DWxF76npUb3Ngw8dKJc7Byzt7rOg4Hhm98TewC8wjGnbVIBKAR3tBfBvx\nqLODhhsHYGABm6Uotld25Wofhiz3so9CdvSLv0d2Wo+w65R500i8c20eJLVyS8gu\nFwAdEYLN2j9qn4A6cs3svCtBAoGBALz0LAuQZkqQb3Xqv01Ux4KmgMxZ68ZQ72Qw\n1lPecxfe+tzx\/tQ7kRrtuQG2wnz3vefwMY21tLZTz1A7jSwgPxtWkkQBt73s7LOA\nERaUbzoE\/VGjNf9p6YFRovjY8O6D6qesfeOz7+rccG8mX+HyhGPBGGEjwqmEoR9S\nlGvxsvb3AoGBAJaPnu+IrZ0AjriemYOX+B1XxIy+g5mkBul537z5yUqvYZobMMk\/\n6TQQjJyUnnh36WRjs11Vr2kIE99YyTCRjljfsPFWnW5Ugt7e9\/G4+L3Kk8FG7RUC\nT1gKxcKcFomctjOS7rVK8\/JxpV7toRhDWkcqMKCTXBnfwW3Iarl1gWuI\n-----END RSA PRIVATE KEY-----\n"
```

Comment:
@zajca tu to nemalo byt skor `cert`? myslim si, ze to netestuje ten klientsky certifikat.

Reply:
... v `doc/docker-compose.yml` bude potrebne spravit novy HTTPS, ktory okrem toho, ze je self-signed bude aj validovat klientske certifikaty ... nemusis to riesit, kedze mas zendesk, upravim to.

Reply:
:facepalm:

Reply:
v pohode. ... teda dokoncim to, ako som pisal.
ta implementacia samotna vyzera byt ok.

Reply:
už sem to změnil :D

Reply:
myslim, ze je potrebne upravit (vytvorit novy) aj ten HTTPS server aby validoval certifikaty .... robis na tom teda dalej, ci to idem doriesit ja?

https://fardog.io/blog/2017/12/30/client-side-certificate-authentication-with-nginx/

Nginx conf:
```
    ssl_client_certificate /etc/nginx/client_certs/ca.crt;
    ssl_verify_client yes;
```

Reply:
Jo máš recht přidal sem ten 142 test jen tak a padlo, to.

Reply:
opat ponukam, ze to dokoncim a kludne sa venuj ZD, ... ale ako chces.

Reply:
OK chtěl sem si to užít, ale asi to teď nestíhám :D možeš to doklepnůt prosím? Ať to zbytečně neblokuju.